### PR TITLE
Removed obfuscated API keys from the Readme and added a screenshot of generated events

### PR DESCRIPTION
**Pull request by codechefnisha-bp**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by codechefnisha-bp** [comment]
@hithwen We received few review comments from datadog account manager. Incorporated those suggestions. 

"API key's shouldn't be on the documentation (although it is obfuscated). As well, the link to the ServiceNow text goes to this URL: https://dev83528.service-now.com/navpage.do, can we change this link? We shouldn't be directing to a dev instance."

Please review the changes. Let us know if anything would be missing.

**Review by ChristineTChen — APPROVED** [review]
LGTM! This can be merged after docs review

File: botprise/README.md [hunk 1]
```diff
@@ -32,10 +35,7 @@ The Botprise integration does not include any service checks.
 All events are sent to the Datadog event stream.
 
 ### Configuration
-To use the Datadog API, you need to enter an [API key and an application key][5]:
-
-Input a Datadog API Key []:xxxx9232ad913d1a864828a2df15xxxx
-Input a Datadog Application Key []:xxxxcb1798718f7a2da141071e7305599d60xxxx
+To use the Datadog API, you need to enter an [API key and an application key][5]
```
Comment (phrawzty) [inline botprise/README.md:38]:
```suggestion
To use the Datadog API, you need to enter an [API key and an application key][5].
```

**Review by phrawzty — APPROVED** [review]
Minor nit, otherwise :shipit: :)

File: botprise/README.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
 ## Overview
 
-Botprise's Datadog integration allows you to send [Botprise][7] events to the event stream in Datadog. It submits Datadog events into the system over email.
+Botprise's Datadog integration allows you to send generated [Botprise][7] events to Datadog using webhook. It helps to monitor your applications and to ensure Botprise is working as expected.
+
+![image-datadog-botprise-events][9]
 
 ## Setup
 
+To use the Botprise-Datadog integration, you must be a customer of Botprise. For more information about Botprise, visit us at <https://www.botprise.com/>.
```
Comment (apigirl) [inline botprise/README.md:9]:
```suggestion
To use the Botprise-Datadog integration, you must be a customer of Botprise. For more information about Botprise, visit us at [https://www.botprise.com/][10].
```

File: botprise/README.md [hunk 3]
```diff
@@ -46,6 +46,7 @@ Need help? Contact [Datadog support][1].
 [3]: https://app.datadoghq.com/infrastructure/map
 [4]: https://docs.datadoghq.com/metrics/
 [5]: https://docs.datadoghq.com/account_management/api-app-keys/
-[6]: https://dev83528.service-now.com/navpage.do
+[6]: https://developer.servicenow.com/dev.do#!/home
 [7]: https://www.botprise.com/
-[8]: https://demoapp.botprise.com/botprise/maindashboard
\ No newline at end of file
+[8]: https://demoapp.botprise.com/botprise/maindashboard
+[9]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/botprise/images/datadog-botprise-events.png
```
Comment (apigirl) [inline botprise/README.md:52]:
```suggestion
[9]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/botprise/images/datadog-botprise-events.png
[10]: https://www.botprise.com/
```

**Review by apigirl — COMMENTED** [review]
the link won't work in our docs unless you actually make it a md link

**Comment by codechefnisha-bp** [comment]
@apigirl Thanks for reviewing. I have added suggested changes. Please approve.

**Pull request merged**